setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
When deciding to use the more efficient list comp vs. for-loop - if there's a side-effect, pick for-loop, otherwise when just accumulating a result, use list-comp.  List comprehensions are very common in Python - it depends on what you're used to looking at.  In other words, what constitutes "readability" depends on the one who reads.

http://stackoverflow.com/questions/2849645/in-python-is-it-better-to-use-list-comprehensions-or-for-each-loops

For this trivial case, I would accept either form.